lib/KBParallel/utils/job.py

```diff
@@ -1,7 +1,8 @@
-from installed_clients.NarrativeJobServiceClient import NarrativeJobService
 from KBParallel.baseclient import BaseClient
 from KBParallel.baseclient import ServerError
 
+from installed_clients.execution_engine2Client import execution_engine2
```

Comment:
is `execution_engine2` a classname? If so, can it be camel uppercase like other classes?

Reply:
That's the name of the client and class. Are you saying to import it as "ExecutionEngine2" or "EE2" or something?